### ISSUE TITLE
Linter Idea: DisableLinterReason

### ISSUE THREAD

**Issue by srawlins** [brigade/scss-lint#514]
This one might be far-fetched, but maybe not. When disabling linters inline, as documented in the [README](https://github.com/brigade/scss-lint/#disabling-linters-via-source), I want to see a comment above the `scss-lint:disable` instruction that explains why individual linter(s) are being disabled via source. Therefore:

**Bad**

```scss
// scss-lint:disable BorderZero
p {
  border: none;
}
```

**Good**

```scss
// This file uses `border: none` for reasons and things. See http://some-discussion.
// scss-lint:disable BorderZero
p {
  border: none;
}
```

One implementation difficulty would be that I think Sass parses these lines into **one** comment node, not two. Hrm...

**Comment by sds** [comment]
The issue you're referring to about multi-line comments being parsed as a single comment is captured here: #373.

As for the linter request: yes, I think this is an odd thing to lint, but happy to merge a pull request.

**Comment by sds** [comment]
Implemented in #519. Thanks!
